ste/sourceInfoProvider-Local_windows.go

```diff
@@ -18,15 +23,24 @@ import (
 
 // This file os-triggers the ISMBPropertyBearingSourceInfoProvider and CustomLocalOpener interfaces on a local SIP.
 
-func (f localFileSourceInfoProvider) Open(path string) (*os.File, error) {
+func (f localFileSourceInfoProvider) GetHandle(path string) (ntdll.Handle, error) {
```

Comment:
1. Nit: This method can be private unless you see a foreseeable need to export it.
2. From the looks of it, I see this function is returning a file descriptor for given path. Also, can you add a small comment to explain the function?